packages/augur-state/src/run.worker.ts

```diff
@@ -0,0 +1,19 @@
+// have to use require in workers so webpack is happy
+const Run = require("./run");
+
+const ctx: Worker = self as any;
```

Comment:
Maybe add a comment here explaining why the typing shenanigans are necessary.

Reply:
Unsure why github isn't picking this up here. I added a comment and pushed.